meadControllerNodeWithWebApp/meadControllerNodeWithWebApp.ino

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Also while not a huge deal, you could factor out your first case and your default case into the a `turnOff` method and then write a `turnOn` method.